entrypoint.sh

```diff
@@ -2,4 +2,10 @@
 
```

Comment:
This script is running without `set -eu` (or something equivalent), could be fixed on this change?

Reply:
Also while we're at it, if you agree, the micrortps_agent command could be exec'd, so there's no unnecessary shell hanging out as the actual payload process's parent